cmd/ttn-lw-cli/commands/end_devices.go

```diff
@@ -1011,7 +1011,7 @@ var (
 					return err
 				}
 				claimInfoResp, err := ttnpb.NewEndDeviceClaimingServerClient(dcs).GetInfoByJoinEUI(ctx, &ttnpb.GetInfoByJoinEUIRequest{
-					JoinEui: devID.JoinEui.Bytes(),
+					JoinEui: devID.GetJoinEui().OrZero().Bytes(),
```

Comment:
This is not ideal due to the fact that a Join EUI of all zeroes is considered valid.

I think it would be better to check for `dev.GetJoinEui() != nil` on line 1007 (in addition to the empty address check).

Reply:
I was under the impression that an JoinEUI fileld with zeros is a valid field but it would be discarded. But reflecting on it, if it was `nil` it really should't even be doing the request in the first place.